scripts/create-agent.ts

```diff
@@ -331,7 +331,7 @@ AWS_DEFAULT_REGION=us-east-1`;
     throw new Error("SSH failed to become ready within timeout (10 minutes). Consider updating to Amazon Linux 2023 AMI for faster boot times.");
   }
 
-  private async executeSSHCommand(keyPath: string, publicDns: string, command: string): Promise<void> {
+  private async executeSSHCommand(keyPath: string, publicDns: string, command: string, tag?: string): Promise<void> {
```

Comment:
the `execSync` in this method needs to update the `stdio` so that each log that is streamed in gets prefixed with the `tag`